### PR TITLE
feature: implement card and images components

### PR DESCRIPTION
**Pull request by sunilvr**
-implement card and images components
-refactored css files

### CODE REVIEW

File: components/buttons/buttons.css [hunk 1]
```diff
@@ -25,38 +25,52 @@
...
 .primary-btn-float{
-    width:auto;
-    border:none;
     padding:0.5rem 0.5rem;
     margin:0.75rem 0.75rem;
 }
 
+.primary-btn-icon{
+    margin:0.5rem 0.5rem;
+    padding: 0 0;
```
Comment (partha8) [inline components/buttons/buttons.css]: here you can have margin:0.5rem, padding: 0

File: components/buttons/buttons.css [hunk 2]
```diff
@@ -25,38 +25,52 @@
...
+.primary-btn-float,
+.primary-btn-icon{
+    width:auto;
+    border:none;
 }
 
 .primary-btn-float{
-    width:auto;
-    border:none;
     padding:0.5rem 0.5rem;
```
Comment (partha8) [inline components/buttons/buttons.css]: padding: 0.5rem can be written

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    border-radius: 1rem;
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  .card-v-header{
+      height:3%;
+      margin:0.3rem 0.2rem;
```
Comment (partha8) [inline components/card/card.css:17]: have margin and padding in multiples of 4, and you can use px here as rem wont have much effects in terms of responsiveness

**Comment by anik31** [comment]
Footer & aside are overlapping while scrolling down, fix that

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    color: var(--text-gray);
+  }
+
+  .card-horizontal {
+    box-shadow: 0 1rem 2rem 0 rgba(0, 0, 0, 0.2);
+    width: 50rem;
+    max-height:40rem;
+    margin: auto;
+    text-align: center;
+    font-family: arial;
```
Comment (anik31) [inline components/card/card.css]: you don't need to add font-family to every element, add it directly to the html

File: components/buttons/buttons.css (hunk 1 above)
Comment (sunilvr) [reply components/buttons/buttons.css]: code change done.

File: components/buttons/buttons.css (hunk 2 above)
Comment (sunilvr) [reply components/buttons/buttons.css]: code change done

File: components/card/card.css (hunk 3 above)
Comment (sunilvr) [reply components/card/card.css:17]: noted. thanks Partha for the review

File: components/card/card.css (hunk 4 above)
Comment (sunilvr) [reply components/card/card.css]: code change done

**Comment by sunilvr** [comment]
> Footer & aside are overlapping while scrolling down, fix that

this has been resolved now Aniket. Thanks for pointing it out

File: components/card/card.css (hunk 3 above)
Comment (sunilvr) [reply components/card/card.css:17]: Hi @partha8 can you kindly approve the PR if its good to be merged?

File: components/card/card.css (hunk 4 above)
Comment (sunilvr) [reply components/card/card.css]: Hi @anik31 can you kindly approve the PR if its good to be merged?

**Review by anik31 — APPROVED** [review]
Good work

**Review by partha8 — APPROVED** [review]

**Pull request merged**